workshops/views.py

```diff
@@ -317,73 +314,6 @@ def person_bulk_add_confirmation(request):
                       context)
```

Comment:
In lines 288 and 309 (views.py) there's still `_verify_upload_person_task` (should be `verify_upload_person_task`).

Why aren't tests catching this?